apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.IntegrationTests/Logging/LoggingTest.cs

```diff
@@ -247,7 +247,7 @@ public async Task Logging_ScopeFormatParameter()
             {
                 await client.GetAsync($"/Main/ScopeFormatParameters/{testId}");
                 var results = _polling.GetEntries(startTime, testId, 1, LogSeverity.Critical);
-                var message = MainController.GetMessage(nameof(MainController.Scope), testId);
+                var message = MainController.GetMessage(nameof(MainController.ScopeFormatParameters), testId);
```

Comment:
We use the name of the method as part of the message to more easily to which test belongs each of the log entries.

Reply:
Yep, copy-paste mistake 😄